### ISSUE TITLE
ML learning material and checklist

### ISSUE THREAD

**Issue by WatershedXiaolan** [WatershedXiaolan/Xiaolan-s-Gitbook#91]
先列一下ML必要知识点：Linear Regression (L1, L2, ElasticNet, Polynomial, Time Series, Seasonal, R square, adj-R, F-statistics), Logistic Regression (Sigmoid, Maximum-Likelihood, Pseudo-R), Tree (CART, Regression Tree, Ensemble, XGB, GBM, AdaBoost, RandomForest, Entropy, Gini, Missing Value, Feature Importance, Pruning, Regularization), SVM (All Kernal), Clustering (K-mean, Hclust, DBSCAN, Distance Metrics, Silhouette Score), Inbalance Classification/Outlier(One Class SVM, LOF, Isolation Forest, Clustering), Missing Value, Feature Engineering Techniques, Dimension Reduction(PCA, Auto-encoder, MDS, LDA), Underfitting & Overfitting.

然后这是optional知识点：NLP (Word2Vec, Sentiment), Neural Network(Optimizer, CNN, LSTM, Tune, Regularization, Gradient Vanishing, Activation), Collaborative Filtering
